README.md

```diff
@@ -199,7 +199,7 @@ whether your [platform](#supported-platforms) is supported or not.
 **Step 2:**
 
 Login your Telegram account first, it needs API ID, API HASH,
-your phone number(in international format) and a login code, the first two
+your phone number(in international format, cannot contain spaces, e.g., `+1234567`) and a login code, the first two
```

Comment:
`+1234567` is not clear in which part is the country code, and which one is the phone number. Would you add more specific notes for it?

Reply:
How about the new commit?